tests/api/test_siblings.py

```diff
@@ -14,291 +14,314 @@
+                    'state_name': 'COMPLETE',
+                    'state_reason': 'All container images have been rebuilt.',
+                    'url': '/api/1/events/1180'
+                },
                 'version': '1.7.4'
             }
         ],
-        'content_types': ['docker'],
-        'created_at': '2017-04-03T14:47:23+00:00',
-        'id': '12327',
-        'issue_date': '2017-08-01T05:59:34+00:00',
-        'product_name': 'Red Hat Enterprise Linux',
-        'product_short_name': 'RHEL',
-        'release_date': None,
-        'reporter': None,
-        'resource_type': 'ContainerAdvisory',
-        'security_impact': 'None',
-        'security_sla': None,
-        'state': 'SHIPPED_LIVE',
-        'status_time': '2017-08-01T15:43:51+00:00',
-        'synopsis': 'cifs-utils bug fix update',
-        'triggered_freshmaker_event': [],
-        'update_date': '2017-08-01T07:16:00+00:00'
-    }])
+        'meta': {
+            'siblings_table_header': 'Nones that triggered ContainerAdvisory 12327'
+        }
+    }),
+    ('containerkojibuild', '710', True, {
+        'data': [
+            {
+                'actual_ship_date': '2017-08-01T15:43:51+00:00',
+                'advisory_name': 'RHBA-2017:2251-03',
+                'assigned_to': None,
+                'attached_bugs': [],
+                'attached_builds': [
+                    {
+                        'completion_time': '2018-04-02T19:39:06+00:00',
+                        'creation_time': '2018-04-02T19:39:06+00:00',
+                        'epoch': '0',
+                        'extra': None,
+                        'id': '811',
+                        'name': 'some_build',
+                        'original_nvr': None,
+                        'release': '4.el7_4_as',
+                        'start_time': '2018-04-02T19:39:06+00:00',
+                        'state': 2,
+                        'version': '1.7.5'},
+                    {
+                        'completion_time': '2017-04-02T19:39:06+00:00',
+                        'creation_time': '2017-04-02T19:39:06+00:00',
+                        'epoch': '0',
+                        'extra': None,
+                        'id': '710',
+                        'name': 'slf4j_2',
+                        'original_nvr': None,
+                        'release': '4.el7_4_as',
+                        'start_time': '2017-04-02T19:39:06+00:00',
+                        'state': 1,
+                        'version': '1.7.4'
+                    }
+                ],
+                'content_types': ['docker'],
+                'created_at': '2017-04-03T14:47:23+00:00',
+                'id': '12327',
+                'issue_date': '2017-08-01T05:59:34+00:00',
+                'product_name': 'Red Hat Enterprise Linux',
+                'product_short_name': 'RHEL',
+                'release_date': None,
+                'reporter': None,
+                'resource_type': 'ContainerAdvisory',
```

Comment:
The `resource_type` returned by the API is `ContainerAdvisory` but the header says that `FreshmakerEvents are returned.`